package.json

```diff
@@ -45,7 +45,7 @@
     "@angular/language-service": "~9.0.5",
     "@types/jasmine": "~3.5.8",
     "@types/jasminewd2": "~2.0.3",
-    "@types/node": "^13.7.7",
+    "@types/node": "^13.9.0",
```

Comment:
@@ -45,7 +45,7 @@
    "@angular/language-service": "~9.0.5",
    "@types/jasmine": "~3.5.8",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "^13.7.7",
    "@types/node": "^13.9.0",

Not working